usb/esp_tinyusb/Kconfig

```diff
@@ -6,6 +6,19 @@ menu "TinyUSB Stack"
         help
             Specify verbosity of TinyUSB log output.
 
+    choice TINYUSB_RHPORT
+        depends on IDF_TARGET_ESP32P4
+        prompt "TinyUSB PHY"
+        default TINYUSB_RHPORT_HS
+        help
+            Based on this parameter the one of two USB PHY will be used for TinyUSB configuration. OTG2.0 PHY for HighSpeed (HS) or OTG1.1 PHY for FullSpeed (FS)
+
+        config TINYUSB_RHPORT_FS
+            bool "FS"
```

Comment:
Nitpick: Could you please also add 'help' section that would contain pin numbers of each PHY?